src/nixpacks/environment.rs

```diff
@@ -86,11 +90,6 @@ mod tests {
         assert!(environment.get_variable("NON_EXISTANT").is_none());
     }
 
-    #[test]
-    fn test_create_invalid_environment() {
-        assert!(Environment::from_envs(vec!["INVALID=ENV=CONFIG"]).is_err());
-    }
-
```

Comment:
Instead of removing this test can you instead assert that it is valid and that the name/value are correct

Reply:
done